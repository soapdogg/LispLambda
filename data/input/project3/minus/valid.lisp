(MINUS 1 13)
(MINUS 13 (MINUS (MINUS 30 23) 7))
(TIMES (MINUS 0 1) 45)
