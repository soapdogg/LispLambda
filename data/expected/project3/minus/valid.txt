-12
13
-45